### DISCUSSION TITLE
AddGlobalNavigationObserver for change of tabs in a tabbed page

### DISCUSSION THREAD

**Discussion by Natpraveen** [PrismLibrary/Prism#2853] [.NET MAUI]
We could see the AddGlobalNavigationObserver observables getting called on creation of the tabs. But when we switch between the tabs the observables are not getting called. 

Believe making the existing observable to fire or having a separate observable for tab changes would help having these information in a single place.

**Comment by dansiegel** [comment]
I've created an issue to track this #2854
